### PR TITLE
feat: #4541 cra template updates

### PR DESCRIPTION
**Pull request by willmcvay**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #4541

### CODE REVIEW

File: packages/cra-template-foundations/template/src/index.tsx [hunk 1]
```diff
@@ -30,8 +30,9 @@ export const renderApp = (Component: React.ComponentType) => {
 
 const run = async () => {
   try {
+    const globalConfig: Config = config
     // Set the global config
-    window.reapit.config = config as Config
+    window.reapit.config = globalConfig
```
Comment (bashleigh) [inline packages/cra-template-foundations/template/src/index.tsx:35]: do we need the delcare here?

**Review by bashleigh — APPROVED** [review]
LGTM! Only one thing I noticed. SVGs now have no EOF line? Guess new prettier conf will fix this?

File: packages/cra-template-foundations/template/src/index.tsx (hunk 1 above)
Comment (willmcvay) [reply packages/cra-template-foundations/template/src/index.tsx:35]: Not sure why but TS didn't like it unless I added the type 🤷🏻

**Comment by willmcvay** [comment]
> LGTM! Only one thing I noticed. SVGs now have no EOF line? Guess new prettier conf will fix this?

Those are auto generated, again, not sure why....

**Pull request merged**